src/main/resources/db/changelog/changes/v1.0.3/sql/create-list-versions-table.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS list_versions
+(
+    id                      UUID            NOT NULL PRIMARY KEY,
+    name                    VARCHAR(255)    NOT NULL,
+    description             VARCHAR(1024),
+    fql_query               text,
+    user_friendly_query     text,
+    fields                  VARCHAR[],
+    updated_by              UUID            NOT NULL,
+    updated_by_username     VARCHAR(1024)     NOT NULL,
+    updated_date            TIMESTAMP       NOT NULL,
+    is_active               boolean         NOT NULL,
+    is_private              boolean         NOT NULL,
+    version                 integer         NOT NULL,
+    list_id                 UUID            NOT NULL,
+    CONSTRAINT fk_report_id FOREIGN KEY (list_id)
+                    REFERENCES list_details (id) MATCH SIMPLE
+);
+ CREATE INDEX IF NOT EXISTS idx_list_id_version ON list_versions (list_id,version);
```

Comment:
when do you see this index being used? the relevant endpoint references `/lists/{list-id}/versions/{version-id}`; I figured this would be the `id` from the table and not `version` (`version` was just for nice human display)

Reply:
@mweaver-ebsco

Reply:
Personal opinion: `{version-id}` in the API should be `version` since it makes the API simpler to use (the URL becomes predictable and simple) without much of a performance hit, while also hiding the version `id` (an implementation detail) behind the version abstraction. In my mind, (list ID, version number) is the conceptual thing that uniquely identifies a version, so the API should reflect that.